packages/calcite-components/src/components/card/card.scss

```diff
@@ -3,18 +3,27 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ *
+ * @prop --calcite-card-accent-color-selected: Specifies the accent color of the component when `selected`.
  * @prop --calcite-card-background-color: Specifies the background color of the component.
  * @prop --calcite-card-border-color: Specifies the border color of the component.
- * @prop --calcite-card-shadow: Specifies the shadow of the component.
+ * @prop --calcite-card-checkbox-background-color-checked: *deprecated* defines the background-color of the sub-component when checked.
+ * @prop --calcite-card-checkbox-background-color: *deprecated* defines the background-color of the sub-component.
+ * @prop --calcite-card-checkbox-icon-color: *deprecated* defines the checkmark color of the sub-component.
+ * @prop --calcite-card-checkbox-shadow-checked: *deprecated* defines the shadow of the sub-component when checked.
+ * @prop --calcite-card-checkbox-shadow-hover: *deprecated* defines the shadow of the sub-component when hovered.
+ * @prop --calcite-card-checkbox-shadow-invalid: *deprecated* defines the shadow of the sub-component when invalid.
+ * @prop --calcite-card-checkbox-shadow: *deprecated* defines the shadow of the sub-component.
+ * @prop --calcite-card-checkbox-size: *deprecated* defines the size of the sub-component.
```

Comment:
I don't think we need to allow the checkbox size to be customized when it's rendered within another component. Applies to Tree Item as well.

Reply:
This was a previously documented token though. I think we need to deprecate and remove in the next breaking change

Reply:
Sorry, I just meant `--calcite-card-checkbox-size` and `--calcite-tree-item-checkbox-size`.